pages/en/lb2/AccessToken-invalidation.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "AccessToken invalidation"
+lang: en
+layout: page
+keywords: LoopBack
+tags: authentication
+sidebar: lb2_sidebar
+permalink: /doc/en/lb2/AccessToken-invalidation.html
+summary: When a user changes their email or password, other sessions are logged out for security reasons.
+---
+
+When user's account is compromised, for example when their password is leaked,
+or the attacker gains access to their email account, the user need means for
```

Comment:
"When a user's account is compromised (for example their password is leaked or an attacker gains access to their email account), the app needs to be able to prevent continued use of the hijacked account."